tests/test_utils/test_moving.py

```
@@ -0,0 +1,261 @@
+            result = moving_window(data, large_window, "median")
+            assert isinstance(result, np.ndarray)
+
+    # Edge cases
+    def test_edge_cases(self, test_data):
+        """Test various edge cases."""
+        edge_cases = [
+            ("single", 1, "Single element array"),
+            ("constant", 3, "Constant values"),
+        ]
+
+        for data_key, window, description in edge_cases:
```

Ruff (0.13.1)

Loop control variable `description` not used within loop body

Rename unused `description` to `_description`

(B007)